commands/actions/publish.go

```diff
@@ -356,3 +354,27 @@ func mustValidateTsconfig(tsconfig *typescript.TsConfig) {
 		os.Exit(1)
 	}
 }
+
+func anyFunctionTsFileExists(actions *actionsModel.ProjectActions) bool {
+	for _, spec := range actions.Specs {
+		locator := spec.Function
+		internalLocator, err := actionsModel.NewInternalLocator(locator)
+		if err != nil {
```

Comment:
Does this check if the file/function exists?

Reply:
No it doesn't. This only check ->  is there two parts when we split string with `:` separator. But we have `mustGetSource` that check file existence.